gui/wxpython/psmap/utils.py

```diff
@@ -361,24 +361,14 @@ def projInfo():
     """Return region projection and map units information,
     taken from render.py
     """
-
-    projinfo = dict()
-
-    ret = RunCommand('g.proj', read=True, flags='p')
-
-    if not ret:
-        return projinfo
-
-    for line in ret.splitlines():
-        if ':' in line:
-            key, val = line.split(':')
-            projinfo[key.strip()] = val.strip()
-        elif "XY location (unprojected)" in line:
-            projinfo['proj'] = 'xy'
-            projinfo['units'] = ''
-            break
-
-    return projinfo
+    proj_info = grass.parse_key_val(
```

Comment:
Thanks for catching this!

This could be shorter if you use grass.parse_command('g.proj', flags='g')

Reply:
> Thanks for catching this!
> 
> This could be shorter if you use grass.parse_command('g.proj', flags='g')

Yes, you are right. I used this code first, but then I found out that this function [`parse_command()`](https://grass.osgeo.org/grass79/manuals/libpython/_modules/script/core.html#parse_command) uses `read_command()` function internally, and according to the text in the [wxGUI submitting code recommendation,](https://trac.osgeo.org/grass/wiki/Submitting/wxGUI) this is "Don't use `grass.run_command()` or `grass.read_command()`. Use functions and classes that use threads like RunCommand". Shouldn't `RunCommand()` class be used in this case?

Reply:
Hmm, you are right, at least somebody is reading the recommendations:) The RunCommand function doesn't use threads though, it's very similar to run/read/write/parse_command, although the error handling works probably better for GUI. So yes,  use RunCommand, you can pass the parse_key_val into RunCommand directly using 'parse' argument, although there is probably no advantage to that.

Reply:
Yes you are right I have already added it (parse arg).